CONTRIBUTING.MD

```diff
@@ -137,4 +137,5 @@ Contributors
 ------------
 *   Pieter Hintjens    
 *   Gareth Hayes    
+*   Ayan Banerjee
```

Comment:
Why did you add your name to this one? You should add it to the AUTHORS file (ok it doesn't exist for this repo, I'll create it) but you should remove it from the contribution protocol itself.

Reply:
OK cool, I'll add the AUTHORS file and ping you when I send a PR.